Paccy10/ampersand_app_api
# Feature(swaps): create swap

**What does this PR do?**

- Create a swap

**Description of Task to be completed?**

- Make a swap

**How should this be manually tested?**

- Checkout to branch `ft-create-swap`
- Run the app with `flask run`
- Send a `POST` request to `/swaps` with the required payload
- A new swap record should be added to the database

**Any background context you want to provide?**

- N/A

**What are the relevant pivotal tracker stories?**

- N/A

**Screenshots (if appropriate)**

- N/A

**Questions:**

- N/A
